geopandas/plotting.py

```diff
@@ -54,6 +54,45 @@ def _flatten_multi_geoms(geoms, prefix="Multi"):
     return components, np.array(component_index)
 
 
+def _expand_kwargs(kwargs, multiindex):
+    """
+    Most arguments to the plot functions must be a (single) value, or a sequence
+    of values. This function checks each key-value pair in 'kwargs' and expands
+    it (in place) to the correct length/formats with help of 'multiindex', unless
+    the value appears to already be a valid (single) value for the key.
+    """
+    from matplotlib.colors import is_color_like
+    from typing import Iterable
+
+    for att, value in kwargs.items():
+        if "color" in att:  # color(s), edgecolor(s), facecolor(s)
+            if is_color_like(value):
+                continue
+        elif att == "offsets":
+            # A single offset is 2-array of numbers.
+            if (
+                isinstance(value, Iterable)
+                and len(value) == 2
+                and pd.api.types.is_number(value[0])
+                and pd.api.types.is_number(value[1])
+            ):
+                continue
```

Comment:
What is the use case for `offsets` here? I am not entirely sure what it does and whether it makes sense to actually support it here. (it is also not tested at the moment)

Reply:
To be honest, I'm not entirely sure. I went through the keywords accepted by `mpl` and made sure additional tests were done on those that are listlike, so that expansion isn't attempted if a valid single value is supplied. If you prefer, we can take it out.

Reply:
Let's remove it for now, since there's likely no application for it within our plotting.

Reply:
alright; i just did that and committed the file